src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4164,6 +4171,19 @@ private bool HasTopLevelNullabilityConversion(TypeWithAnnotations source, TypeWi
                     break;
 
                 case ConversionKind.ImplicitNullable:
+                    if (trackMembers && node.Kind == BoundKind.Conversion)
```

Comment:
Consider using an `is` check instead of the kind and conversion.

Reply:
These checks should be removed soon.

---
In reply to: [269237234](https://github.com/dotnet/roslyn/pull/34303#discussion_r269237234) [](ancestors = 269237234)